lua/plugins/configs/navic.lua

```diff
@@ -0,0 +1,39 @@
+local present, navic = pcall(require, "nvim-navic")
+
+if not present then
+   return
+end
+
+local icons = require "plugins.configs.lspkind_icons"
+
+local options = {
+   depth = 5,
+
+   icons = {
+      ["class-name"] = "%#NavicItemKindClass#" .. icons.Class .. "%*" .. " ",
```

Comment:
For example, `["class-name"]` should be changed to `Class` (according to Navic doc)